src/components/search.js

```diff
@@ -81,14 +80,17 @@ export const Search = ({ onFocusChange }) => {
   };
 
   return (
-    <ParentDiv onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave} onFocus={handleMouseEnter}>
+    <ParentDiv
```

Comment:
I think you can rename parentdiv and searchdiv to better names, they don't match the quality of your code anymore